ci/tests/arakoon/extend_promote_demote_test.py

```diff
@@ -447,3 +449,48 @@ def ovs_3671_validate_archiving_of_existing_arakoon_data_on_create_and_extend_te
             for filename in files_to_create:
                 assert client.file_exists(filename) is False, 'File {0} is missing'.format(filename)
             TestArakoon.verify_arakoon_structure(client, cluster_name, False, False)
+
+    @staticmethod
+    def ovs_4509_validate_arakoon_collapse_test():
+        """
+        Validate arakoon collapse
+        """
+        node_ips = [sr.ip for sr in GeneralStorageRouter.get_storage_routers()]
+        node_ips.sort()
+        arakoon_conf_file = '/etc/init/ovs-arakoon-ovsdb.conf'
+        etcd_config = 'etcd://127.0.0.1:2379/ovs/arakoon/ovsdb/config'
+        tlog_location = '/opt/OpenvStorage/db/arakoon/ovsdb/tlogs'
+        first_ip = node_ips[0]
```

Comment:
If you want to retrieve the ip of the node running the test you can use:
from ovs.extensions.generic.system import System
ip = System.get_my_storagerouter().ip